example/app/data/DemoDataSource/DemoPackage/TabsPluginConfig.json

```diff
@@ -5,18 +5,23 @@
     {
       "attributeType": "boolean",
       "type": "dmss://system/SIMOS/BlueprintAttribute",
-      "name": "addChildTabsOnRender",
+      "name": "childTabsOnRender",
       "contained": true,
-      "optional": true,
-      "default": false
+      "optional": false
```

Comment:
Makes sense, though then we should update TabsContainer, right, so they align? I removed the optional flag because the parameters were not made optional in TTabsPluginConfig.